Cakes-Co/index.html

```diff
@@ -2,15 +2,65 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Responsive Cake webpage</title>
+  <link rel="stylesheet" href="/style.css" />
 </head>
 
 <body>
-    <!-- Add your markup here -->
+  <div class="container">
+    <header>
+      <div class="logo"></div>
+      <div class="menu">
+        <div class="open" id="open"></div>
+      </div>
+      <nav id="nav" class="nav-close">
+        <ul class="list">
+          <li class="nav-link"><a href="#">HOME</a></li>
+          <li class="nav-link"><a href="#">CAKES</a></li>
+          <li class="nav-link"><a href="#">ORDERIN</a></li>
+          <li class="nav-link"><a href="#">LESSON</a></li>
+          <li class="nav-link"><a " href=" #">ABOUT</a></li>
+        </ul>
+      </nav>
+      <div class="overlay"></div>
+      <div class="content">
+        <p class="content-p">The best cakes in town deliverd to door.</p>
+      </div>
+    </header>
+
+    <main>
+      <section class="welcome">
+        <div class="card">
```

Comment:
could you have used a semantic tag here instead of div?

Reply:
I could use an article for that. If I am not wrong.